clients/privacy-center/public/fides-consent-demo.html

```diff
@@ -1,62 +1,75 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <title>fides-consent script demo page</title>
 
     <script src="/fides-consent.js"></script>
 
     <style>
-      body {
-        max-width: 800px;
-        margin: 16px;
-      }
-      pre {
-        background-color: #eee;
-        padding: 16px;
-      }
+        body {
+            max-width: 800px;
+            margin: 16px;
+        }
+        
+        pre {
+            background-color: #eee;
+            padding: 16px;
+        }
     </style>
-  </head>
+</head>
 
-  <body>
+<body>
     <main>
-      <h1>fides-consent.js demo page</h1>
-      <p>
-        This page exists to demonstrate and debug the fides-consent.js script.
-        It is used by the Privacy Center's test suite to ensure the a users's
-        consent choices can be accessed by pages that live outside of the
-        Privacy Center.
-      </p>
-
-      <h2>Was the fides global defined?</h2>
-      <pre id="has-fides"></pre>
-
-      <h2>Consent JSON</h2>
-      <pre id="consent-json"></pre>
+        <h1>fides-consent.js demo page</h1>
+        <p>
+            This page exists to demonstrate and debug the fides-consent.js script. It is used by the Privacy Center's test suite to ensure the a users's consent choices can be accessed by pages that live outside of the Privacy Center.
+        </p>
+
+        <h2>Was the fides global defined?</h2>
+        <pre id="has-fides"></pre>
+
+        <h2>Consent JSON</h2>
+        <pre id="consent-json"></pre>
+
+        <h2>Consent Banner Options</h2>
+        <pre id="consent-banner-options"></pre>
     </main>
-  </body>
+</body>
 
-  <script>
+<script>
     (() => {
-      const hasFides = typeof Fides === "object";
-      document.getElementById("has-fides").textContent = String(hasFides);
-
-      if (!hasFides) {
-        return;
-      }
-
-      // Pretty-print the fides consent object and add it to the page.
-      document.getElementById("consent-json").textContent = JSON.stringify(
-        Fides.consent,
-        null,
-        2
-      );
-
-      // Test behavior of integrations that can be configured without/before their platform scripts.
-      Fides.gtm();
-      Fides.meta({
-        consent: Fides.consent.tracking,
-        dataUse: Fides.consent.data_sales,
-      });
+        const hasFides = typeof Fides === "object";
+        document.getElementById("has-fides").textContent = String(hasFides);
+
+        if (!hasFides) {
+            return;
+        }
+
+        // Initialize the banner, providing any test options (if injected by test code)
+        Fides.banner(window.fidesConsentBannerOptions);
```

Comment:
This is the new part - by default, `window.fidesConsentBannerOptions` is `undefined`, but we set it in the Cypress tests to configure what gets injected into this `Fides.banner()` call at startup. This allows testing different kinds of options.